components/Tabs/src/TabBarView/MDCTabBarView.h

```diff
@@ -28,6 +52,9 @@ __attribute__((objc_subclassing_restricted)) @interface MDCTabBarView : UIScroll
 /** The color of the Tab bar's background. */
 @property(nullable, nonatomic, copy) UIColor *barTintColor;
 
+/** The tab bar view delegate. */
+@property(nullable, nonatomic, weak) id<MDCTabBarViewDelegate> delegate;
```

Comment:
Let's name this `tabBarDelegate` so we don't require the two delegates to be the same.

Reply:
Done.